rackcity/utils/change_planner_utils.py

```diff
@@ -178,3 +178,117 @@ def get_page_count_response_for_cp(request, change_plan):
     count = assets.count() + assetsCP.count()
     page_count = math.ceil(count / page_size)
     return JsonResponse({"page_count": page_count})
+
+
+def get_changes_on_asset(asset, assetCP):
+    fields = [field.name for field in Asset._meta.fields]
+    changes = []
+    for field in fields:
+        if getattr(asset, field) != getattr(assetCP, field):
+            # TODO will need to serialize field if it's another model
+            changes.append({
+                "field": field,
+                "live": getattr(asset, field),
+                "change_plan": getattr(assetCP, field),
+            })
+    return changes
+
+
+def get_cp_modification_conflicts(asset_cp):
+    conflicts = []
+    nonresolvable_message = "This conflict cannot be resolved; the change " + \
+        "needs to be removed from your change plan."
+    conflicting_asset_message_1 = "Due to more recent live changes, " + \
+        "your change planner modification to this asset's "
```

Comment:
small nit but maybe instead of modifications, say `your change plan's version of this's assets: ` because a conflict could come from a created asset on cp.

Reply:
cool I like